### DISCUSSION TITLE
пример как добавить строку

### DISCUSSION THREAD

**Discussion by Romanufa999** [totumonline/totum-ru-issues-and-discussions#609] [Как это сделать?]
Добрый день. как добавить строку через remote без PRO
в видео только как с строку с одним значением , а как передать и разобрать если передается строка где 5 знаxений в столбце ? без PRO &

**Comment by ttm-alexeypolunin** [comment]
В коде remote в том числе в MIT:

```
=: insert(table: 'some_table'; field: 'field_1' = $#post[key_1]; field: 'field_2' = $#post[key_2])
```

что значит `если передается строка где 5 знаxений в столбце` непонятно
